### PR TITLE
[Twig][Bridge] force space between widget and label in checkbox_radio_label

### PR DESCRIPTION
**Pull request by MatTheCat**
| Q | A |
| --- | --- |
| Fixed tickets | N/A |
| License | MIT |

Update

without space

to

with space

### CODE REVIEW

**Comment by MatTheCat** [comment]
I don't know anything about xpath but I think it's not normal tests fail because of one space.

**Comment by stof** [comment]
@MatTheCat it is normal, because this XPath query asserts the text of the label, and you now have a space in it which was not there before

**Comment by MatTheCat** [comment]
Should we use `normalize-space` then?

**Comment by xabbuh** [comment]
I think you should add the space in the test. If we used `normalize-space`, we would not be able to detect future regressions.

**Comment by MatTheCat** [comment]
I don't get why tests are failing past PHP5.5 =/

**Comment by stof** [comment]
@xabbuh thus it would not work. ``normalize-space`` will change multiple spaces to 1, but there would still be a difference between 1 space and none

@MatTheCat we have an issue with the deps=low and deps=high tests. @nicolas-grekas is currently working on fixing them.

**Comment by fabpot** [comment]
:+1:

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 1]
```diff
@@ -168,8 +168,7 @@
             {% set label = name|humanize %}
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
-            {{- widget|raw -}}
-            {{- label is not sameas(false) ? label|trans({}, translation_domain) -}}
+            {{- widget|raw }} {{ label is not sameas(false) ? label|trans({}, translation_domain) -}}
```
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: Should be `same as` now that `sameas` is deprecated.

Comment (MatTheCat) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: Should this be made in this PR?

**Comment by fabpot** [comment]
Thank you @MatTheCat.

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: Never mind, this has been fixed in the meantime in 99a1fcc3e2d51a9be176ad5d8ed1fcda4a4492d7.

**Pull request closed without merging**